packages/components/modules/profiles/ProfilePopover/AddProfileModal/index.tsx

```diff
@@ -0,0 +1,179 @@
+import { FC } from 'react'
+
+import { CloseIcon, TextField } from '@baseapp-frontend/design-system'
+import { setFormRelayErrors } from '@baseapp-frontend/utils'
+
+import { zodResolver } from '@hookform/resolvers/zod'
+import { LoadingButton } from '@mui/lab'
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Link,
+  Typography,
+} from '@mui/material'
+import { useForm } from 'react-hook-form'
+import { ConnectionHandler } from 'react-relay'
+import slugify from 'slugify'
+
+import { useOrganizationCreateMutation } from '../../graphql/mutations/OrganizationCreate'
+import { schema } from './constants'
+import { Form } from './styled'
+import { AddProfileModalProps, OrganizationCreateForm } from './types'
+
+let nextClientMutationId = 0
+
+const AddProfileModal: FC<AddProfileModalProps> = ({
+  addNewProfileLabel = 'New organization',
+  termsAndConditionsUrl = '',
+  addNewProfileDescription = 'Create an organization and invite multiple members to manage and collaborate.',
+  submitLabel = 'Create Organization',
+  onClose,
+  open,
+  setOpen,
+  userId,
+}) => {
+  const form = useForm<OrganizationCreateForm>({
+    defaultValues: {
+      name: '',
+      urlPath: '',
+    },
+    resolver: zodResolver(schema),
+  })
+
+  const [commitMutation, isMutationInFlight] = useOrganizationCreateMutation()
+
+  const handleClose = () => {
+    setOpen(false)
+    form.reset()
+    onClose?.()
+  }
+
+  const onSubmit = (data: OrganizationCreateForm) => {
+    if (isMutationInFlight) return
+
+    nextClientMutationId += 1
+    const clientMutationId = nextClientMutationId.toString()
+
+    const connections = ConnectionHandler.getConnectionID(userId, 'ProfilesListFragment_profiles')
+
+    commitMutation({
+      variables: {
+        input: {
+          name: data.name,
+          urlPath: data.urlPath,
+          clientMutationId,
+        },
+        connections: [connections],
+      },
+      onCompleted: (response, errors) => {
+        if (errors) {
+          console.error(errors)
+          return
+        }
+        const mutationErrors = response?.organizationCreate?.errors
+        setFormRelayErrors(form, mutationErrors)
+
+        if (!mutationErrors?.length) {
+          handleClose()
+        }
+      },
+      onError: console.error,
+    })
+  }
+
+  return (
+    <Dialog
+      open={open}
+      onClose={handleClose}
+      aria-labelledby="organization-modal-title"
+      aria-describedby="organization-modal-description"
+      PaperProps={{
+        sx: {
+          borderRadius: { xs: 0, sm: 2 },
+          minWidth: { xs: '100%', sm: 0 },
+          margin: { xs: 0, sm: 2 },
+          maxWidth: { sm: 366 },
+          height: { xs: '100%', sm: 'auto' },
+          minHeight: { xs: '100%', sm: 'auto' },
```

Comment:
is that specified in the designs? I'm asking because we do have a dialog component under `packages/design-system/components/dialogs/Dialog` so if that's a style that applies to all our dialogs, we should consider doing these changes on that component